project/Build.scala

```diff
@@ -173,6 +173,10 @@ object MacroGLBuild extends MechaRepoBuild {
       "-optimise",
       "-feature"
     ),
+    libraryDependencies ++= Seq(
```

Comment:
Removing for now - could've been a problem with another sbt lwjgl plugin version I was using.